### ISSUE TITLE
Feature Request: Configurable "dunstctl close" Behavior

### ISSUE THREAD

**Issue by S-u-b** [dunst-project/dunst#846]
According to dunstctl's help output:
```
$ dunstctl --help
  close                             Close the *last* notification
```

It's unclear in which sense "last" is being used here, until one reads the man page:
```
$ man 1 dunstctl
...
       close
           Close the *topmost* notification currently being displayed.
...
```

I'd like to propose new global option being made available to control whether the "first" vs "last" or "topmost" vs "bottommost" notification is closed: "close_order" (_or go with whatever you prefer_).  Once implemented, if generating notifications as follows:
`notify-send 1`
`notify-send 2`
`notify-send 3`
`notify-send 4`
`notify-send 5`

...produces these notification results:
1
2
3
4
5

Then setting the **close_order** option to "top" or "last" closes the oldest notification when `dunstctl close` is run:
**~~1~~**
2
3
4
5

While setting the **close_order** option to "bottom" or "first" closes the oldest notification when `dunstctl close` is run:
1
2
3
4
**~~5~~**

**Comment by fwSmit** [comment]
Good point! This will be relevant when #845 is implemented, since the topmost notification may not be the last one.
I would probably go with the name `close_priority` with the options of `top`, `bottom`, `old(est)`, `new(est)`.

**Comment by vincentbernat** [comment]
Related, it could be convenient to filter by date too. I want to close notifications older than 1 hour for example. I was looking at doing it myself, but it seems there is no way to list current notifications or get their characteristics. Adding a NotificationListDisplayed, NotificationListWaiting, NotificationClose (with ID) DBus calls would help.

**Comment by bynect** [comment]
dunstctl close now has an optional ID arg.
for listing open notifications there is work going on here: https://github.com/dunst-project/dunst/pull/1348
